src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -112,6 +114,9 @@ export const SelectInstitutionForm = ({
             </Paper>
           )}
           <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+            <Typography variant="h3" sx={{ fontWeight: 'normal' }}>
+              {t('common.select_institution')}
+            </Typography>
```

Comment:
Moved this into the form instead of outside as previously, since we already have a similar header on the other field